.github/workflows/stale.yml

```
@@ -1,18 +1,18 @@
-name: 'Close stale issues and PR'
+name: Close stale issues and PR
 on:
   schedule:
-    - cron: '30 1 * * *'
+  - cron: 30 1 * * *
```

```suggestion
  - cron: '30 1 * * *'
```
Un-apply this fix since `*` is a wildcard character.